default.nix

```diff
@@ -1,20 +1,47 @@
-{nixpkgs ? import <nixpkgs> {}, compiler ? "ghc865"}:
+{ compiler ? "ghc865" }:
+
 let
-  dapptools = builtins.fetchGit {
-    url = "https://github.com/dapphub/dapptools.git";
-    rev = "c44db1f252b870c0913d0f83093347e9c05469c9";
+  # git ls-remote https://github.com/NixOS/nixpkgs-channels.git nixos-19.09
```

Comment:
any reason to not go latest here? I know dapptools is on 19.09 but if 20.03 works with the overrrides it might be preferable

Reply:
version conflicts: https://ipfs.io/ipfs/QmcrDwY3Y3DPT1t58qzhfG6XWUuKSw5Ajtq6RQ8Qf3qicG/log.txt

```
 This package indirectly depends on multiple versions of the same package. This is very likely to cause a compile failure.
```

Reply:
I guess that it's getting confused because `hevm` still uses `19.09` even when we bump to `20.03` here, which is causing multiple versions of the same package to be used?

Reply:
Gotcha. Yeah, using 19.09 seems good then